src/pages/products/turn/connecting.astro

```diff
@@ -0,0 +1,181 @@
+        >TURN GPS Bill Pay system</a
+      > to access the online payment portal. The cost is $600 and gives you access to TURN GPS Network also. You'll need
+      a credit card to process your payment.
+    </p>
+    <ul>
+      <li>● Select “Register” on the navigation pane on the left-hand side of the page.</li>
+      <li>
+        ● Click the “New Registration” button. (Note: Both new subscriptions and renewed subscriptions for Nevada GPS
+        are considered “new” to the TURN system.)
+      </li>
+      <li>● Review the Terms and Conditions, and then click “I agree” to continue.</li>
+      <li>
+        ● Fill out all the required fields, including the name of your organization, your login/username, a password for
+        your TURN GPS account, and your personal contact information.
+      </li>
+      <li>● Click “Register” to create your account.</li>
+    </ul>
+    <p>
+      At this point, the system will direct you to the payment page. Make sure that you have your payment information
+      readily available, as the system will automatically time out after a while, requiring you to start over with a
+      whole new login/username, password, and email. If the system keeps coming back with the password field empty, the
+      system may think that your email is already in use and, therefore, will not allow you to move forward. Each
+      subscription needs a unique email address (even if it is a made-up one [e.g., gps_login@BobsBigBurger.com]).
+    </p>
+    <p>
+      After you have created/renewed and paid for your subscription, you will receive an email with a receipt. It
+      typically takes the system about 24 hours to process a payment and activate an account for fieldwork using the
+      TURN GPS Network.
+    </p>
+    <div class="mx-auto w-full max-w-4xl">
+      <YouTube
+        class="aspect-video max-w-full rounded-lg border-2 border-black shadow-xl"
+        id="T4Q2eqrrbXQ"
+        playlabel="TURN Bill Pay System tutorial video"
+      />
+    </div>
+  </Section>
+  <Section {...page.section[2]}>
+    <p>
+      The TURN GPS Network, which serves Utah and some areas of surrounding states with 113 sensors. We are currently in
+      the process of expanding this network in Utah, Idaho, and Wyoming. The TURN Network runs on the Trimble Pivot
+      Platform. You may Login using your User Name and Password. If you don't have an account yet, you can register <a
+        href="https://turngps-billpay.ugrc.utah.gov/">here</a
+      >. If you haven't already, you will need to create a Utah ID.
+    </p>
+    <p>
+      TURN GPS is a statewide network of permanently located GPS receivers, which provides both real time corrections
+      and data for post processing to those that have an active subscription. Current cost is $600 for each User Login
+      and is good for a full year.
+    </p>
+    <ul>
+      <li><b>The Coordinate Reference Frame for this network is NAD83(2011)(EPOCH:2010.0000.)</b></li>
```

Comment:
Use `<strong>` instead of `<b>` tags.

From the docs...

> ... you should not use <b> for styling text or granting importance. If you wish to create boldface text, you should use the CSS [font-weight](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight) property. If you wish to indicate an element is of special importance, you should use the [<strong>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong) element.